src/test/testWeekDaysPart2/testWeekDaysPart2.java

```diff
@@ -0,0 +1,52 @@
+package testWeekDaysPart2;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import weekDaysPart2.WeekDaysWithFiles;
+
+import java.io.IOException;
+import java.net.URL;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.fail;
+import static weekDaysPart2.WeekDaysWithFiles.*;
+
+public class testWeekDaysPart2 {
+
+    @Test
+    public void testUsual() {
+        String filePath = getFilePath("number_of_days.txt");
+        try {
+            WeekDaysWithFiles.read(filePath);
+        } catch (FileFormatException | IOException e) {
```

Comment:
@kolegova-mary do not think that it is a correct approach with try/catch in test. I would suggest to remove them from the body and add 'throws' for both possible exceptions